src/integrations/close.io/routes.js

```diff
@@ -60,12 +59,14 @@ module.exports = function (app, model, Implementation, opts) {
 
   function customerLead(request, response) {
     new CloseioCustomerLeadGetter(
-      Implementation, _.extend(request.query, request.params),
-      opts, integrationInfo,
+      Implementation,
+      _.extend(request.query, request.params),
+      opts,
+      integrationInfo,
     )
       .perform()
       .then((lead) => {
-        if (!lead) { throw { status: 404, message: 'not_found' }; }
+        if (!lead) { throw new Error('not_found'); }
```

Comment:
I did a the change because the linter wants an error to be thrown

The `catch` below the `throw` didn't use the `status`/ `message` for the error handling.
So it won't break anything from my point of view.